volto/src/addons/volto-govuk-theme/src/customizations/volto/components/theme/Forbidden/Forbidden.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * @module components/theme/Forbidden/Forbidden
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Container } from 'semantic-ui-react';
+import { withServerErrorCode } from '@plone/volto/helpers/Utils/Utils';
+
+/**
+ * forbidden function.
+ * @function Forbidden
+ * @returns {string} Markup of the forbidden page.
+ */
+const Forbidden = () => (
+  <div class="app-width-container">
+    <main
+      class="govuk-main-wrapper govuk-main-wrapper--l"
+      id="main-content"
+      role="main"
+    >
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-9">Forbidden</h1>
+          <p className="govuk-body">
+            We apologize for the inconvenience, but you don't have permissions
+            on this resource.
+          </p>
+          <p className="govuk-body">Thank you.</p>
+        </div>
+      </div>
+    </main>
+  </div>
+);
+
+export default withServerErrorCode(403)(Forbidden);
```

Comment:
There are security reasons why we actually shouldn't tell people it's a 403 when it is. Our public users should get 404s for any 403s, so they don't know that there's something at that URL that just hasn't been published yet.

We could use a Forbidden page specifically within the CMS views - so we only show this to someone signed in who has viewer permissions but not for this particular resource. Might need to go back to UCD/product to check the scope